app/ceom/maps/views.py

```diff
@@ -2,7 +2,7 @@
 from django.http import HttpResponse
 from django.contrib.auth.decorators import login_required
 from django.contrib.gis.geos import Polygon
```

Comment:
Put the polygon and point import on the same line with a comma between Polygon, Point